reference/enterprise-requirements.rst

```diff
@@ -58,5 +59,37 @@ applies:
 |                       |                       | services              |
 +-----------------------+-----------------------+-----------------------+
 
-[note] **Note:** Three nodes are required for multi-node operation.
-[/note]
+.. note ::
+    Three nodes are required for multi-node operation.
+
+Role Based Minimum Memory Sizing
+--------------------------------
+
+For multi-node deployments the following minimum component memory sizing may be
+used to size each node based on the roles it hosts within the deployment:
+
++-----------------+----------+---------+--------------+----------+--------+-----------------+
+| Component       | Control  | Compute | Storage      | Sunbeam  | Client | Juju-Controller |
```

Comment:
In my view, this would work better if node types would be represented as rows and hosted services would be represented as columns. Feels more natural for me when screening the table.